src/XMLimport.h

```diff
@@ -118,6 +118,14 @@ class XMLimport : public QXmlStreamReader
     int module;
     int mMaxRoomId;
     int mMaxAreaId; // Could be useful when iterating through map data
+
+    static const int            cmTriggerView;
```

Comment:
Well it is not the safest of code constructs but actually it is not wise to reimplement something that already exists in the `dlgTriggerEditor` class already.  It would be better to move the corresponding `static const` members from `private` to `public` in `dlgTriggerEditor.h` and then `#include "dlgTriggerEditor.h` in the collection of includes at the top of `XMLimport.cpp` and then you do not need to duplicate them (with the exact same values) in the latter class - all you then need to do is add a `dlgTriggerEditor::` prefix to the uses in the `XMLimport.cpp` file.  I have rolled these changes up as https://github.com/dicene/Mudlet/pull/9 for you @dicene...